packages/components/typography/src/Caption/Caption.tsx

```diff
@@ -1,18 +1,16 @@
 import React from 'react';
 import { FontWeightTokens } from '@contentful/f36-tokens';
 import type {
-  CommonProps,
-  MarginProps,
   PolymorphicComponent,
   PolymorphicProps,
   ExpandProps,
 } from '@contentful/f36-core';
-import { Text } from '../Text';
+import { Text, TextProps } from '../Text';
```

Comment:
Could you please add the type prefix to all `TextProps` imports you've made?

```suggestion
import { Text, type TextProps } from '../Text';
```